garrysmod/lua/vgui/dscrollpanel.lua

```diff
@@ -5,20 +5,23 @@ AccessorFunc( PANEL, "Padding", "Padding" )
 AccessorFunc( PANEL, "pnlCanvas", "Canvas" )
 
 function PANEL:Init()
-
 	self.pnlCanvas = vgui.Create( "Panel", self )
-	self.pnlCanvas.OnMousePressed = function( self, code ) self:GetParent():OnMousePressed( code ) end
+	self.pnlCanvas.OnMousePressed = function( pnl, code )
+		pnl:GetParent():OnMousePressed( code )
+	end
+
 	self.pnlCanvas:SetMouseInputEnabled( true )
 	self.pnlCanvas.PerformLayout = function( pnl )
-
 		self:PerformLayoutInternal()
 		self:InvalidateParent()
-
 	end
 
 	-- Create the scroll bar
-	self.VBar = vgui.Create( "DVScrollBar", self )
-	self.VBar:Dock( RIGHT )
+	local scrollBar = vgui.Create( "DVScrollBar", self )
+	if IsValid( scrollBar ) then

```

Comment:
Code formatting style is inconsistent with the rest of the repository. (Many more cases of this not commented on that should also be changed)

Reply:
Well, if it's so important, it's not hard for me to do a style match, I'll do it in the next commit